### ISSUE TITLE
ORM Agnostic

### ISSUE THREAD

**Issue by victorbstan** [elevation/event_calendar#35]
How hard is it to make this plugin DB agnostic? Does it have to rely on ActiveRecord?

**Comment by jeffschuil** [comment]
I don't think it would be too difficult, though I haven't personally tried a nosql db. I've noticed a couple forks where people removed or modified the 'ActiveRecord::Base.extend EventCalendar::ClassMethods' line in railtie.rb, which I assume is for this purpose.

It seems to me you just need to get the events from the database (based on their start and end fields), and then the rest should work the same. This is done in the events_for_date_range method.
